package.json

```diff
@@ -18,7 +18,7 @@
     "lint": "eslint --ext js,jsx,ts,tsx .",
     "lint:md": "remark . -o",
     "test": "lerna run --concurrency 1 test",
-    "build": "lerna run --parallel build",
+    "build": "lerna run --scope cozy-device-helper build && lerna run --parallel --ignore cozy-device-helper build",
```

Comment:
I am wondering what have changed inside cozy-intent to get this error. We need to investigate that point one day. Do you think it diserve a Github issue? 🤔

Reply:
pourquoi pas oui 👍